tests/test_provisioning.py

```diff
@@ -221,3 +221,21 @@ def test_reprovision_provider(monkeypatch, db_session, client, test_app):
     ]
     assert len(provisioned_providers) == 1
     assert provisioned_providers[0]["type"] == "prometheus"
+
+
+@pytest.mark.parametrize(
```

Comment:
- maybe add a test where `KEEP_DASHBOARDS` is not a valid json?
- maybe add a test where app reloads more than once?